test/test_page.py

```diff
@@ -1,17 +1,51 @@
 # coding=utf-8
 from __future__ import unicode_literals
 
+import sys
 import json
 import codecs
 from test.constants import RANGE_COLOR, CLOTHES, WEEK
 from pyecharts import (
     Bar, Scatter3D, Line, Pie, Map, Kline, Radar, WordCloud, Liquid
 )
 from pyecharts import Page
-from nose.tools import eq_, raises
+from nose.tools import eq_, assert_list_equal, raises
 
 TEST_PAGE_TITLE = "my awesome chart"
 
+PY36 = sys.version_info >= (3, 6)
+
+
+class MockChart:
+    def __init__(self, page_title):
+        self.page_title = page_title
+
+
+def test_page_init():
+    line = MockChart('Line-Chart')
```

Comment:
from mock import MagicMock

line = MagicMock(page_title="Line-Chart")
# for the rest of the MockChart, do the same. so MockChart is not required.